docs/tutorials/customizing-the-output/custom-parameterization-of-helm-charts-kustomize-octemplates.md

```diff
@@ -41,7 +41,7 @@ In this example, we illustrate how we could parameterize a custom field in the h
           name: orders
       spec:
           progressDeadlineSeconds: 600
-          replicas: {{ index .Values "common" "replicas" }}
+          replicas: {% raw %}{{ index .Values "common" "replicas" }}{% endraw %}
```

Comment:
Might be better to move the raw block to the start and end of the yaml?

Reply:
tried that. But for some strange reason, this particular block of yaml becomes messed up if the raw block extends end to end of the yaml.

Reply:
hmm.. okay